doc/source/tune/api_docs/schedulers.rst

```diff
@@ -3,65 +3,25 @@
 Trial Schedulers (tune.schedulers)
 ==================================
 
-In Tune, some hyperparameter optimization algorithms are written as "scheduling algorithms". These Trial Schedulers can early terminate bad trials, pause trials, clone trials, and alter hyperparameters of a running trial.
+In Tune, some hyperparameter optimization algorithms are written as "scheduling algorithms".
+These Trial Schedulers can early terminate bad trials, pause trials, clone trials,
+and alter hyperparameters of a running trial.
 
-All Trial Schedulers take in a ``metric``, which is a value returned in the result dict of your Trainable and is maximized or minimized according to ``mode``.
+All Trial Schedulers take in a ``metric``, which is a value returned in the result dict of your
+Trainable and is maximized or minimized according to ``mode``.
 
 .. code-block:: python
 
     tune.run( ... , scheduler=Scheduler(metric="accuracy", mode="max"))
```

Comment:
We should recommend that the user passes `metric` and `mode` to `tune.run` instead

Reply:
@Yard1 going forward I think we should even go as far as to disallow this behaviour. Sometimes it can be good to have several ways of doing the same thing, here I think it provides no value and is confusing.

Reply:
I actually disagree, you may want to use one metric for the Searcher and another for Scheduler. That being said if you are using the same one for both defining it in tune.run is definitely the best option